ReferencePluginC/ControlC.cs

```diff
@@ -64,8 +64,14 @@ public void VerseRefChanged(IPluginChildWindow sender, IVerseRef oldReference, I
 
 		public void NextVerse(Object sender, EventArgs e)
 		{
-			m_Reference = m_Reference.GetNextVerse(m_Project);
-
+			var versification = m_Project.Versification;
+			int bbbcccvvv = 0;
+			do
+			{
+				m_Reference = m_Reference.GetNextVerse(m_Project);
+				bbbcccvvv = m_Reference.BBBCCCVVV;
+			} while (versification.IsExcluded(bbbcccvvv));
```

Comment:
It might be worth creating an overload of GetNextVerse that skips over excluded verses. Paratext already includes it as a bool internally so it would be a simple thing to add.